e2e_samples/parking_sensors/scripts/configure_databricks.sh

```diff
@@ -73,12 +73,14 @@ photon_node_types=$(echo "$node_types" | jq -r '.node_types[] | select(.photon_d
 # Find common VM sizes
 common_vms=$(grep -Fwf <(echo "$photon_node_types") vm_names.txt)
 
-# Find the VM with the least resources
+# Find the least resource option that has at least 4 times the memory compared to the number of cores
```

Comment:
It might be more clear to say "4GB memory for each core" instead of "4 times the memory compared to the number of cores"